docs/platform_concepts/growth/metrics.md

```diff
@@ -109,19 +109,34 @@ This widget shows the funnel of users who are coming to the bot. The blue lines
 
 ----
 
-### Whatsapp consumption
+### WhatsApp comsumption
 
-> This is calculated for WhatsApp channel.
+In the WhatsApp comsumption widget, you can select a WhatsApp phone number from the dropdown to see the consumption for that specific number. If no number is selected, the data will be displayed for all the numbers combined.
```

Comment:
Just a suggestion:

If no number is selected, the data will be compiled and displayed for all the numbers collectively.